docs/source/beginner-tutorial.rst

```diff
@@ -96,6 +96,20 @@ To now test this algorithm on financial data, ``zipline`` provides three
 interfaces: A command-line interface, ``IPython Notebook`` magic, and
 :func:`~zipline.run_algorithm`.
 
+Ingesting Data
+^^^^^^^^^^^^^^
+If you haven't ingest the data, run:
```

Comment:
Sorry to slow this down - could you just fix the typo: "ingest" should be "ingested", and we'll merge this.  Really appreciate the doc improvements here.